e2e/Makefile

```diff
@@ -337,3 +338,40 @@ daemonset-lvmd/test: topolvm.img $(GINKGO) $(HELM) $(KUBECTL)
 
 .PHONY: daemonset-lvmd/clean
 daemonset-lvmd/clean: daemonset-lvmd/delete-minikube daemonset-lvmd/remove-vg
+
+.PHONY: embedded-lvmd/create-vg
+embedded-lvmd/create-vg: daemonset-lvmd/create-vg
+
+.PHONY: embedded-lvmd/setup-minikube
+embedded-lvmd/setup-minikube: daemonset-lvmd/setup-minikube
+
+.PHONY: embedded-lvmd/launch-minikube
+embedded-lvmd/launch-minikube: daemonset-lvmd/launch-minikube
+
+.PHONY: embedded-lvmd/delete-minikube
+embedded-lvmd/delete-minikube: daemonset-lvmd/delete-minikube
+
+.PHONY: embedded-lvmd/test
+embedded-lvmd/test: topolvm.img $(GINKGO) $(HELM) $(KUBECTL)
```

Comment:
The only difference between `daemonset-lvmd/XXX` and `embedded-lvmd/XXX` is the `values.yaml` file path, so I want to remove the duplication.

My idea is as follows:

- Rename `daemonset-lvmd/foo` target to something like `incluster-lvmd/foo` so that it can be used for both daemonset lvmd and embedded lvmd installation.
- Remove `embedded-lvmd/foo`.
- Remove `HELM_VALUES_FILE_LVMD_EMBEDDED` variable.
- In github workflow, change the value of `HELM_VALUES_FILE_LVMD` via command line option of `make` command.

Then, when we want to run e2e test for daemonset lvmd, we can do it as follows.
```
make -C e2e incluster-lvmd/create-vg
make -C e2e incluster-lvmd/setup-minikube
make -C e2e incluster-lvmd/launch-minikube
make -C e2e incluster-lvmd/test
```

When we want to run e2e test for embedded lvmd, we can do it as follows.
```
make -C e2e incluster-lvmd/create-vg  # Same as daemonset lvmd
make -C e2e incluster-lvmd/setup-minikube  # Same as daemonset lvmd
make -C e2e incluster-lvmd/launch-minikube  # Same as daemonset lvmd
make -C e2e incluster-lvmd/test HELM_VALUES_FILE_LVMD = manifests/values/embedded-lvmd-storage-capacity.yaml
```

@satoru-takeuchi 
I would also like to hear from you about my idea. What do you think?

Reply:
Good idea. I will use a github matrix to extend the tests via env variable so we can also get rid of the additional github actions file. I think this is definetly a big improvement.

Reply:
Changed it so that the file is changed based on the matrix, also took over your suggestion for `incluster-lvmd`